src/widgets/lv_btnmatrix.c

```diff
@@ -23,7 +23,7 @@
 #define MY_CLASS &lv_btnmatrix_class
 
 #define BTN_EXTRA_CLICK_AREA_MAX (LV_DPI_DEF / 10)
```

Comment:
Instead of `LV_DPI_DEF` this should be used:
```suggestion
#define BTN_EXTRA_CLICK_AREA_MAX ((lv_disp_get_dpi(lv_obj_get_disp(obj)) / 10)
```

`lv_disp_get_dpi()` return `lv_coord_t` so the `U` postfix is not required.

Reply:
Done in [1533507](https://github.com/lvgl/lvgl/pull/3241/commits/153350792b364cf3978b92df2a00c73721119acb)